code/modules/power/apc.dm

```diff
@@ -267,8 +267,6 @@
 		update_state |= UPSTATE_WIREEXP
 	if(!update_state)
 		update_state |= UPSTATE_ALLGOOD
-
-	if(update_state & UPSTATE_ALLGOOD)
 		if(emagged)
```

Comment:
Shouldn't the indentation of this also change?

Reply:
No, the results are the same since that only happens if there was no other update_state flag on.